Xamarin.MacDev/XamMacSdk.cs

```diff
@@ -16,13 +16,13 @@ namespace Xamarin.MacDev
 {
 	public sealed class XamMacSdk : IMonoMacSdk
 	{
-		static readonly MacOSXSdkVersion[] MacOSXSdkVersions = {
-			MacOSXSdkVersion.V10_7,
-			MacOSXSdkVersion.V10_8,
-			MacOSXSdkVersion.V10_9,
-			MacOSXSdkVersion.V10_10,
-			MacOSXSdkVersion.V10_11,
-			MacOSXSdkVersion.V10_12
+		static readonly AppleSdkVersion [] MacOSXSdkVersions = {
+			AppleSdkVersion.V10_7,
+			AppleSdkVersion.V10_8,
+			AppleSdkVersion.V10_9,
+			AppleSdkVersion.V10_10,
+			AppleSdkVersion.V10_11,
+			AppleSdkVersion.V10_12
```

Comment:
more missing versions
should not it be identical to the previous one ?

Reply:
Same.